src/Components/Cart/Cart.js

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from 'react';
+import Product from './Product/Product';
+import './Cart.scss';
+
+const Cart = () => {
+  const [productData, setProductData] = useState([]);
+
+  let totalSumPrice = 0;
+
+  const deletedList = e =>
+    setProductData(
+      productData.filter(product => product.id !== Number(e.target.id))
+    );
+
+  const goToPay = () =>
+    fetch('api주소', {
+      method: 'POST',
+      headers: {
+        'content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        productData: productData,
+      }),
+    });
+
+  const productList = productData.map((product, idx) => {
+    totalSumPrice += product.price * product.count;
+    return (
+      <Product
+        key={idx}
+        id={product.id}
+        idx={idx}
+        name={product.name}
+        size={product.size}
+        count={product.count}
+        price={product.price}
+        deletedList={deletedList}
+        productData={productData}
+        setProductData={setProductData}
+      />
+    );
```

Comment:
```suggestion
    return (
      <Product
        key={idx}
        product={product}
        idx={idx}
        deletedList={deletedList}
        productData={productData}
        setProductData={setProductData}
      />
    );
```
product 객체에 있는 값들은 한 번에 보내주면 어떨까요>?

Reply:
무조건 구조 분해 할당한다고 좋은 게 아니었네요. 
막상 코드 칠 때는 몰랐는데 이렇게 보니 어지럽네요 ㅎㅎ,,
리뷰 반영했습니다.!